### PR TITLE
File Added : Assignment-9

### PR DESCRIPTION
**Pull request by roshnidurugwar**

### CODE REVIEW

File: Roshni/html/Assignment9/Assignment9.html [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+            <hr />
+        </div>
+        <div class="header2 ">
+            <ul>
+                <li>
+                    &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<span style="color: black"><h2 style="position: absolute;top: 20px;left: 300px; font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif">TiMES</span>
+                    <span style="color: red">JOBS</h2></span>
+                    <span style="font-size: x-large"></span>
+                </li>
+                <li><span style="position: absolute;left: 500px;top: 20px;"><h4 style="background-color:orange;opacity: 60%;">Is your Career affected by covid-19?&nbsp;&nbsp;<input type="submit" value="YES" style="background-color: red;width: 60px;color: white;height: 30px;border-radius: 13%;">&nbsp;&nbsp;<input type="submit" value="NO" style="background-color: orange; width: 60px;height: 30px;border-radius: 13%;"></h4></span></li>
```
Comment (Priyankashelke2411) [inline Roshni/html/Assignment9/Assignment9.html]: give space after :

File: Roshni/html/Assignment9/Assignment9.html [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+                    </div>
+                </div>
+            </div>
+            &nbsp;&nbsp;&nbsp; &nbsp;&nbsp;&nbsp;
+            <div class="column">&nbsp;&nbsp;&nbsp; <img class="rightimgcss" src="user-engage.gif" alt="times" /><br /><br /></div>
+            &nbsp;&nbsp;&nbsp; &nbsp;&nbsp;&nbsp;
+            <div class="column">&nbsp;&nbsp;&nbsp; <img class="rightimgcss" src="Screenshot_2021-05-20 Jobs Search Recruitment Employment Job Vacancies TimesJobs.png" alt="times" /><br /><br /></div>
+        </div>
+    </body>
+</html>
```
Comment (Priyankashelke2411) [inline Roshni/html/Assignment9/Assignment9.html:128]: add extra line at the end

**Review by Priyankashelke2411 — CHANGES_REQUESTED** [review]

File: Roshni/html/Assignment9/Assignment9.html (hunk 1 above)
Comment (roshnidurugwar) [reply Roshni/html/Assignment9/Assignment9.html]: i have made changes maam

File: Roshni/html/Assignment9/Assignment9.html (hunk 2 above)
Comment (roshnidurugwar) [reply Roshni/html/Assignment9/Assignment9.html:128]: i have made changes maam

**Review by Oshin-250197 — APPROVED** [review]

**Review by arshad05786 — APPROVED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**